README.md

```diff
@@ -311,12 +311,29 @@ We use flask-babel to do internationalisation. To extract messages from source a
 make translation-templates
 ```
 
+make translation-templates is a command that runs:
```

Comment:
I don't think this is correct. make translation-templates runs `pybabel extract` and only once to generate a single static messages.pot (which is the same for all languages). Then it uses eq-translations to extract a schema specific .pot template. 

Even so, I think this is going into more detail than is needed for the readme. Probably enough to say 'make translation templates is a command that uses pybabel to extract static messages and eq-translations to extract schema messages' or similar. If a dev is interested in the exact command they can dig into the script itself.